.idems_app/deployments/plh/skins/index.ts

```diff
@@ -0,0 +1,27 @@
+import { IAppSkin } from "data-models";
+
+// PLH-specific skins
+
+/** Skins can be applied to an app to override the core APP_CONFIG.
```

Comment:
This comment can be moved to the data-models definition. if you put `@example` in the comment it should give nice output when hovering the definition